openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -1276,13 +1275,28 @@ def test_register_form_third_party_auth_running_google(
                     "value": "",
                     "default": False
                 }
+            ] + [
```

Comment:
Sounds like this is not necessity in my opinion. This is just test code that we shouldn't care about much as long as it's passing.